ophyd/signal.py

```diff
@@ -74,8 +74,17 @@ def trigger(self):
         '''Call that is used by bluesky prior to read()'''
         # NOTE: this is a no-op that exists here for bluesky purposes
         #       it may need to be moved in the future
+
```

Comment:
Now that all the reporting is done through the Status objects, do we need any of the changes to this file?

Reply:
correct I have removed these changes

Reply:
correct, they have been removed

Reply:
I thought I found them all, I will take another look.